app/components/page-components/Chatbot/ChatbotModTools.vue.ts

```diff
@@ -19,6 +18,8 @@ export default class ChatbotModTools extends ChatbotBase {
     this.chatbotApiService.fetchSymbolProtection();
     this.chatbotApiService.fetchLinkProtection();
     this.chatbotApiService.fetchWordProtection();
+
+    // this.chatbotCommonService.bindsToasted(this.$toasted);
```

Comment:
Let's remove this if it's no longer needed.  If there's a reason to leave it in here but commented out, then let's leave a comment explaining what it is and why someone would uncomment it.